pkg/api/v20170930/types.go

```diff
@@ -0,0 +1,16 @@
+package v20170930
+
+// VersionInfo contains rersion and release numbers
+type VersionInfo struct {
+	Release string `json:"release,omitempty"`
+	Version string `json:"version"`
+}
+
+// OrchestratorInfo contains orchestrator version info
+type OrchestratorInfo struct {
+	Orchestrator string `json:"orchestrator"`
+	VersionInfo
```

Comment:
If Orchestrator is a separate field does VersionInfo need to a struct? In general the organization of this struct is confusing. Why result orchestratorProfile here?